civihr_default_theme/scss/pages/_admin_absence_tab.scss

```diff
@@ -1,47 +1,68 @@
-.chr_leave-absences-admin {
-  background: transparent;
-  box-shadow: none;
-  padding: 0;
-}
-
-.chr_absence-tab-page {
-
+.absence-tab-page {
   .button-container {
     margin-bottom: 20px;
   }
 
   .tabs-container {
+    .container {
+      /*to shift the container to left else it was not aligned with button*/
+      padding-left: 0;
+    }
 
     ul {
-      margin-left: 0px;
+      margin-left: 0;
+      /**/
       background: white;
 
       li {
-
         &:first-child {
+          /*to match the shift from left in the design*/
           margin-left: 20px;
         }
 
         a {
           font-size: 14px;
+          cursor: pointer;
+          /*otherwise it jumps in on hover*/
+          border-width: 0px;
+
+          &:hover {
+            /*as by default it sets it to #fbfbfc*/
+            background-color: transparent;
+            border-width: 0px;
```

Comment:
if i remove this is what i get 
![image](https://cloud.githubusercontent.com/assets/89973/25090854/4dcad300-23a3-11e7-9a60-35185933e3ed.png)